spi/src/main/java/org/arquillian/cube/spi/metadata/HasMappedPorts.java

```diff
@@ -0,0 +1,13 @@
+package org.arquillian.cube.spi.metadata;
+
+
+public interface HasMappedPorts extends CubeMetadata {
+    
+    String getIP();
+
+    Integer forTargetPort(int targetPort);
```

Comment:
If this were expanded to replace Binding, I think that would be appropriate.  That said, replacing Binding was a lot more work.  I'll look again and see if I can make that work, but it's a pretty significant refactoring effort.